### PR TITLE
OC-11208 Add site OID, site ID and site name information in response of get Participants webservice

### PR DESCRIPTION
**Pull request by theywa**
https://jira.openclinica.com/browse/OC-11208

### CODE REVIEW

**Comment by svadla-oc** [comment]
jenkins retest this please

File: web/src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java [hunk 1]
```diff
@@ -481,7 +481,16 @@ public String startParticipantImportJob(String studyOid, String siteOid, HttpSer
...
 		            	
 		            	ArrayList<StudyParticipantDTO> studyParticipantDTOs = getStudyParticipantDTOs(studyOid, siteOid,study);            	  		 	            
-		 	            responseSuccess.setStudyParticipants(studyParticipantDTOs);		 	          
+		 	            responseSuccess.setStudyParticipants(studyParticipantDTOs);
+		 	            responseSuccess.setSiteOID(siteOid);
+		 	            if (siteOid != null) {
+							StudyBean site = this.getStudyDAO().findByOid(siteOid);
+							responseSuccess.setSiteID(site.getId());
+							responseSuccess.setSiteName(site.getName());
+						} else {
```
Comment (svadla-oc) [inline web/src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java]: If siteOid is null, you should not set any values at all. Leave them as nulls

Comment (theywa) [reply web/src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java]: code updated

**Comment by svadla-oc** [comment]
👍

**Pull request merged**